decidim-core/spec/commands/decidim/create_registration_spec.rb

```diff
@@ -95,6 +95,23 @@ module Comments
             expect { command.call }.to change(User, :count).by(1)
           end
 
+          it "downcases the user nickname" do
+            expect(User).to receive(:create!).with(
+              name: form.name,
+              nickname: "username",
```

Comment:
Should be like this, right?

```suggestion
              nickname: "nickname",
```

Reply:
No, it is not. The nickname is not a form attribute, but it is a method that calls `Decidim::UserBaseEntity.nicknamize(name, organization:)` having as parameter the name attribute.

Reply:
And here https://github.com/decidim/decidim/blob/22391289175b28940b5dcf3a7c9df041cc0648bc/decidim-core/spec/commands/decidim/create_registration_spec.rb#L84 ? 

I understand that this spec is a leftover from #11510 - right? So shall we fully remove the nickname from the form params and change this new example to "what happens when the name has some uppercase values?"

Reply:
(It works as expected BTW) 
![image](https://github.com/user-attachments/assets/e30a574b-2e56-472e-a475-a7aff08b128a)

Reply:
fixed in [f791b95](https://github.com/decidim/decidim/pull/14272/commits/f791b95f4e20122eb2f2c8e5d564b58267d180f2)